mitxgraders/comparers/comparers.py

```diff
@@ -112,30 +113,108 @@ class EqualityComparer(Comparer):
     True
     >>> grader(None, '[1/sqrt(2), 0, 1/sqrt(2)]')['ok']
     True
-    >>> FormulaGrader.reset_default_comparer()
+    >>> MatrixGrader.reset_default_comparer()
+
+    FormulaGrader and MatrixGrader both have set_default_comparer() and
+    reset_default_comparer() methods.
 
     """
     schema_config = Schema({
         Required('transform', default=None): Nullable(is_callable)
     })
 
-    def __call__(self, comparer_params_eval, student_eval, utils):
-        expected_input = comparer_params_eval[0]
-
+    @staticmethod
+    def validate(expected_eval, student_eval, utils):
         if hasattr(utils, 'validate_shape'):
             # in numpy, scalars have empty tuples as their shapes
-            shape = tuple() if isinstance(expected_input, Number) else expected_input.shape
+            shape = tuple() if isinstance(expected_eval, Number) else expected_eval.shape
             utils.validate_shape(student_eval, shape)
 
-        # If provided, apply the transform function
-        if self.config['transform'] is not None:
-            expected_input = self.config['transform'](expected_input)
-            student_eval = self.config['transform'](student_eval)
+    def transform(self, evaluation):
+        f = self.config['transform']
+        return evaluation if f is None else f(evaluation)
+
 
-        return utils.within_tolerance(expected_input, student_eval)
+    def __call__(self, comparer_params_eval, student_eval, utils):
+        expected_eval = comparer_params_eval[0]
+        self.validate(expected_eval, student_eval, utils)
+
+        expected_eval = self.transform(expected_eval)
+        student_eval = self.transform(student_eval)
+
+        return utils.within_tolerance(expected_eval, student_eval)
 
 equality_comparer = EqualityComparer()
 
+class MatrixEntryComparer(EqualityComparer):
+    """
+    Default comparer for MatrixGrader. Compares student and instructor matirx
+    evaluations entry-by-entry for equality.
+
+    Configuration
+    =============
+
+        transform (None | function): same as EqualityComparer, defaults to None
```

Comment:
Just bringing style in line with other such entries:
`Same as EqualityComparer (default None)`
Blank line afterwards.